src/Sylius/Bundle/AddressingBundle/.gitignore

```diff
@@ -7,6 +7,7 @@
 /test/app/cache
 /test/app/logs
 /test/var/cache
+/test/var/db.sql
```

Comment:
@loic425 it seems you wanted to avoid committing `db.sql` but the file is commited already 😉